lib/gameComponents/deck.dart

```diff
@@ -1 +1,30 @@
-class Deck {}
+import 'card.dart';
+import 'hand.dart';
+
+class Deck {
+  List<Hand> hands = [];
+  List<Card> cards = [];
+
+  Deck() {
+    for (var suit in Suit.values) {
+      for (var rank in ranks.entries) {
```

Comment:
nit: these properties are never mutated, so these could be made `final` instead of `var`